src/mantine-core/src/components/Skeleton/demos/usage.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Skeleton } from '../Skeleton';
+
+const code = `
+<Skeleton height={45} />
+<Skeleton height={45} animated />
+<Skeleton height={45} type='circle' />
```

Comment:
Make sure that demo code has same formatting as the rest of the code (incorrect quote types, should be double)